src/app/content/hooks/locationChange/resolveContent.ts

```diff
@@ -132,8 +132,9 @@ const resolvePage = async(
   const pageId = match.state && 'pageUid' in match.state
     ? match.state.pageUid
     : getPageIdFromUrlParam(book, match.params.page);
+  const bookConfig = getBookVersionFromUUIDSync(book.id);
 
-  if (!pageId) {
+  if (!pageId || bookConfig?.retired) {
```

Comment:
i think we should move this into resolveBook, you'll have to change `getIdFromPageParam` to not need a specific page, which isn't used anyway